.htmltest.yml

```diff
@@ -1,6 +1,6 @@
 DirectoryPath: "public"
 IgnoreURLs:
   - "https://members.nottinghamtec.co.uk"
+  - "https://static.nottinghamtec.co.uk/*"
```

Comment:
Static is my bad, my branch was from an old master where the video was being loaded from there, and I had forgot that we removed the .htaccess from members